demos/benchmark/lv_demo_benchmark.c

```diff
@@ -47,13 +56,27 @@
 #define FALL_HEIGHT     80
 #define PAD_BASIC       8
 
+
+#define LV_TEST_FONT_STRING_1 "法律之前人人平等,并有权享受法律的平等保护,不受任何歧视。人人有权享受平等保护,以免受违反本宣言的任何歧视行为以及煽动这种歧视的任何行为之害"
```

Comment:
`LV_TEST_FONT_STRING_CHINESE`
It is recommended to use meaningful names, and we may introduce tests in other languages, such as Arabic, in the future.